Brushing README (#86)
openshift/node-observability-operator: README.md

# NodeObservability Operator

**Note**: This Operator is in the early stages of implementation and keeps changing.

> The NodeObservability Operator allows you to deploy and manage [NodeObservability Agent](https://github.com/openshift/node-observability-agent) on worker nodes. The agent is deployed through DaemonSets on all or selected nodes.     

Reviewer: ```suggestion
The NodeObservability Operator allows you to deploy and manage [NodeObservability Agent](https://github.com/openshift/node-observability-agent) on the worker nodes. The NodeObservability agent is deployed through DaemonSets on all or selected nodes.     
```

> Afterward, the profiling requests can be created to trigger different types of profiling. The profiling data would be accessible in the root filesystem of the NodeObservability Agent.

Reviewer: ```suggestion
Afterward, you can create the profiling requests to trigger different types of profiling. The profiling data will be accessible in the root filesystem of the NodeObservability Agent.
```

> The profiling requests can be created after to trigger different types of profiling. The profiling data is available on the root filesystem of the NodeObservability Agent for later retrieval.

Reviewer 2: ```suggestion
Afterward, the profiling requests can be created to trigger different types of profiling. The profiling data would be accessible in the root filesystem of the NodeObservability Agent.
```

- [Deploying the  NodeObservability Operator](#deploying-the-nodeobservability-operator)
    - [Installing the NodeObservability Operator](#installing-the-nodeobservability-operator)
    - [Installing the NodeObservability Operator using a custom index image on the OperatorHub](#installing-the-nodeobservability-operator-using-a-custom-index-image-on-the-operatorhub)
- [Verifying the installation of NodeObservability Operator](#verifying-the-installation-of-nodeobservability-operator)
- [Creating the NodeObservability CR](#creating-the-nodeobservability-cr)

## Deploying the `NodeObservability` Operator

> You can deploy the NodeObservability Operator by using one of the two below procedures.

Reviewer 2: ```suggestion
NodeObservability Operator can be deployed using one of the following procedures.
```

1. To build and push the Operator image into a registry, run the following commands:
   ```sh
   # set the envar CONTAINER_ENGINE to the preffered container manager tool (default is podman)
   export IMG=${REGISTRY}/${REPOSITORY}/node-observability-operator:${VERSION}
   make container-build container-push
   ```
2. To deploy the NodeObservability Operator, run the following command:
    ```
    make deploy
    ```

>     If you want to specify the agent image of your choice, patch the operator deployment with the following command:

Reviewer 2: ```suggestion
   To specify the custom image for the NodeObservability Agent, patch the deployment of the operator with the following command:
```

    ```sh
    oc set env deployment/node-observability-operator --containers=manager RELATED_IMAGE_AGENT=${MY_IMAGE_AGENT} -n node-observability-operator
    ```

### Installing the `NodeObservability` Operator using a custom index image on the OperatorHub
**Note**: It is recommended to use `podman` as a container engine.

> You can install the NodeObservability Operator by building and pushing the custom OLM bundle index image.

Reviewer 2: ```suggestion
NodeObservability Operator can be installed by building and pushing the custom OLM bundle index image.
```

   EOF
   ```

> 6. Install the operator from the command line client or from the webUI (down below):

Reviewer: ```suggestion
6. Install the operator from the command line client or from the webUI:
```
